tests/results/3eadb018d45336f73e6c0f620de84057eb2ffb214f0deb699434aaa01e64a28d/result.json

```diff
@@ -269,7 +269,7 @@
     "extracted": [
       {
         "name": "extracted_wscript.bat",
-        "sha256": "0435a4727d2a8f71ecdd5b510cec45e05bf4fbeba42b1eb80655da8ccea81c6e"
```

Comment:
If you did check that the content is basically the same, no problem, else it would be good to validate.